ci/ray_ci/macos/macos_ci_build.sh

```diff
@@ -49,4 +49,43 @@ build_x86_64() {
   if [ "$BUILDKITE_BRANCH" = "master" ]; then python .buildkite/copy_files.py --destination jars --path ./.jar/darwin; fi
 }
 
+build_aarch64() {
+  # Cleanup environments
+  rm -rf /tmp/bazel_event_logs
+  cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
+  (which bazel && bazel clean) || true
+  brew install pkg-config nvm node || true
+  # TODO(simon): make sure to change both PR and wheel builds
+  # Special setup for jar builds (will be installed to the machine instead)
+  # - brew remove --force java & brew uninstall --force java & rm -rf /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask
+  # - brew install --cask adoptopenjdk/openjdk/adoptopenjdk8
+  diskutil list external physical
+  export JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home
+  java -version
+  # Build wheels
+  export UPLOAD_WHEELS_AS_ARTIFACTS=1
+  export MAC_WHEELS=1
+  export MAC_JARS=1
+  export RAY_INSTALL_JAVA=1
+  export RAY_ENABLE_WINDOWS_OR_OSX_CLUSTER=1
+  export MINIMAL_INSTALL=1
```

Comment:
This line is added; without this, the script tries to install full dependencies which fails (it's also failing in master now). The build and test wheel only needs minimal dependencies.